baybe/constraints/discrete.py

```diff
@@ -278,20 +286,36 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         return data.index[mask_bad]
 
 
-# the order in which the constraint types need to be applied during discrete subspace
-# filtering
+@define
+class DiscreteCardinalityConstraint(CardinalityConstraint, DiscreteConstraint):
+    """Class for discrete cardinality constraints."""
+
+    # class variables
```

Comment:
Typons in capitalization in these two comments.

Reply:
Fixed.